public/Day_1/style.css

```
@@ -0,0 +1,217 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)